torch_audiomentations/utils/dsp.py

```diff
@@ -0,0 +1,32 @@
+import librosa
+import numpy as np
+import torch
+
+
+def calculate_rms(samples):
+    """
+    Calculates the root mean square.
+
+                Taken from https://github.com/iver56/audiomentations/blob/master/audiomentations/core/utils.py
+    """
+    return torch.sqrt(torch.mean(torch.square(samples), -1, keepdims=True))
+
+
+def calculate_desired_noise_rms(clean_rms, snr):
+    """
+    Given the Root Mean Square (RMS) of a clean sound and a desired signal-to-noise ratio (SNR),
+    calculate the desired RMS of a noise sound to be mixed in.
+    Based on https://github.com/Sato-Kunihiko/audio-SNR/blob/8d2c933b6c0afe6f1203251f4877e7a1068a6130/create_mixed_audio_file.py#L20
+                Taken from https://github.com/iver56/audiomentations/blob/master/audiomentations/core/utils.py
```

Comment:
Which of those two?

Reply:
Actually, audiomentations took it from audio-SNR. We can keep the audio-SNR link and remove the audiomentations one.